relayer/workers/beefyrelayer/main.go

```diff
@@ -135,18 +139,6 @@ func (worker *Worker) Start(ctx context.Context, eg *errgroup.Group) error {
 	return nil
 }
 
-func (worker *Worker) Stop() {
```

Comment:
we don't need this anymore?

Reply:
Nope. was never being called. Riz and I decided that workers shouldn't have a Stop function, and instead they should just clean up gracefully themselves after encountering an error.